src/map/script_constants.hpp

```diff
@@ -3203,7 +3203,7 @@
 	export_constant(EFST_SPELLFIST);
 	export_constant(EFST_NETHERWORLD);
 	export_constant(EFST_SIREN);
-	export_constant(EFST_HANDICAPSTATE_DEEP_SLEEP);
```

Comment:
Should we export these as deprecated to still have "support" but also print out the deprecation message?